### PR TITLE
Added dropdowns

### PR DESCRIPTION
**Pull request by cshinn**
Addresses #8 

Summary of changes: Added the dropdown basic as well as dropdown states and dropdown styles

### CODE REVIEW

File: source/_patterns/basics/dropdown/00-dropdown.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+title: Dropdown
+---
+This is a *Markdown* description of my pattern.
```
Comment (andresgalante) [inline source/_patterns/basics/dropdown/00-dropdown.md]: Please add link to bootstrap docs following the template

File: source/_patterns/basics/dropdown/02-dropdown-states.mustache [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{{> basics-dropdown(dropdownText: "Closed Dropdown") }}
+{{> basics-dropdown(dropdownText: "Open Dropdown", dropdownState: "open") }}
+<br>
```
Comment (andresgalante) [inline source/_patterns/basics/dropdown/02-dropdown-states.mustache]: Why is this here?

Comment (cshinn) [inline source/_patterns/basics/dropdown/02-dropdown-states.mustache]: Dropdowns need to be triggered manually in bs4, so I put these here so people could see both versions. Is there a way to add JS to a basic? If so, I'll do it that way instead

File: source/_patterns/basics/dropdown/00-dropdown.md (hunk 1 above)
Comment (cshinn) [inline source/_patterns/basics/dropdown/00-dropdown.md]: yeah, I realized that I forgot this. Adding it now, then will update

File: source/_patterns/basics/dropdown/02-dropdown-states.mustache (hunk 2 above)
Comment (andresgalante) [inline source/_patterns/basics/dropdown/02-dropdown-states.mustache]: I meant the `<br>` can we avoid them?

**Comment by cshinn** [comment]
@andresgalante I've addressed the comments you made and beefed up the documentation

File: source/_patterns/basics/dropdowns/00-dropdown.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+---
+## Overview
+
+This is a Bootstrap component, [see complete documentation here](http://v4-alpha.getbootstrap.com/components/dropdowns/)
+
+Patternfly design specs are at [http://www.patternfly.org/pattern-library/widgets/#dropdowns](http://www.patternfly.org/pattern-library/widgets/#dropdowns)
+
+Must add a modifier class to add color to the label.
+
+## Accessibility
```
Comment (andresgalante) [inline source/_patterns/basics/dropdowns/00-dropdown.md]: You are missing content here

**Comment by cshinn** [comment]
ok @andresgalante I think it's finally ready!

**Comment by andresgalante** [comment]
👍 yay! thanks @cshinn

**Comment by srambach** [comment]
LGTM
